src/Connection.lua

```diff
@@ -46,8 +46,8 @@ end
 --   or nil in case there was an error executing the statement
 -- @treturn table|nil in case there was an error executing the statement or nil if the statement
 --   was executed successfully
--- [impl -> dsn~luasql-connection-execute~0]
 function Connection:execute(statement)
+    -- [impl -> dsn~luasql-connection-execute~0]
```

Comment:
Moving the coverage tag inside the function hides it from the generated API documentation.